src/Bart/Optional/Present.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Bart\Optional;
+
+/**
+ * Class that handles containing an existent (non-null) value
+ * Class Present
+ * @package Bart\Optional
+ */
+class Present extends Optional
```

Comment:
I felt like Present was a better name, since either the value is there or it isn't. "Some" would also work, it just seemed to make what was going on a little less clear when reading through the code.